rio/srcrobot/subsystems/indexer.py

```diff
@@ -34,7 +34,7 @@ def indexerIntakeOnce(self):
             .finallyDo(lambda interrupted: self.stopIndexer()).withName("IntakeOnce")
 
     def indexerShoot(self): 
-        return self.run(lambda: self.indexerMotor.set(TalonSRXControlMode.Velocity, self.indexerShootVelocity)).withName("Shoot")
+        return self.run(lambda: self.indexerMotor.set(TalonSRXControlMode.Velocity, self.indexerShootVelocity)).until(lambda: not self.getBeamBreakState()).withTimeout(1.0).finallyDo(self.stopIndexer()).withName("Shoot")
```

Comment:
This will stop the indexer once the beam break clears. That is not what we want. We should andThen to a wait seconds to keep it running until the note completely clears.